tests/evhtests/l7_evh_graph_test.go

```diff
@@ -194,13 +198,15 @@ func TestNoBackendL7ModelForEvh(t *testing.T) {
 
 	modelName, _ := GetModelName("foo.com", "default")
 	SetUpTestForIngress(t, modelName)
+	objects.SharedAviGraphLister().Delete(modelName)
+	fmt.Printf("HAHA: modelName: %s", modelName)
```

Comment:
please remove all fmt prints which are not required.

Reply:
removed, thanks